components/playgroundChatbot.tsx

```diff
@@ -354,10 +354,7 @@ export default function PlaygroundChatbot(props: {
                 functionJsonResponse &&
                 typeof functionJsonResponse === "object"
               ) {
-                contentString = convertToRenderable(
-                  functionJsonResponse,
-                  `${functionNameToDisplay(chatItem?.name ?? "")} result`
-                );
+                contentString = chatItem.content;
```

Comment:
You missed one champ

Reply:
don't understand this comment 😆

Reply:
This change was in the last PR I reviewed :smile:

Reply:
Ohhh you're right :facepalm: 

Muppet of the day award goes to me :joy: